src/zulip.rs

```diff
@@ -195,7 +195,7 @@ fn handle_command<'a>(
                     next = words.next();
                 }
 
-                Ok(Some(String::from("Unknown command")))
+                Ok(Some(String::from("Unknown command, check triagebot's documentation at <https://forge.rust-lang.org/triagebot/notifications.html>")))
```

Comment:
Are the brackets around the link needed?

Reply:
It's kinda the default formatting for links over the web, it's just a way to signify it. It isn't necessary for Zulip to parse it as a clickable link.

If it's too ugly I'll just modify it :)

Reply:
Why does this link to the notifications page? There are other features that accept Zulip commands, and I'm concerned it could be confusing for the user since it could imply that notifications are the only commands it accepts.

Reply:
I was having problems finding the notifications page, so I didn't notice that that implementation is the one that also handles meetings. Should be fixed now (I added another message for manual triggers @ing triagebot)